common/src/commonMain/kotlin/com/cuhacking/atlas/common/SearchViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.cuhacking.atlas.common
+
+import com.cuhacking.atlas.db.Database
+import com.cuhacking.atlas.db.Feature_fts
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+class SearchViewModel(private val database: Database) {
+
+    private val _searchResults: MutableStateFlow<List<SearchResult>> = MutableStateFlow(listOf())
```

Comment:
```suggestion
    private val _searchResults: MutableStateFlow<List<SearchResult>> = MutableStateFlow(emptyList())
```
I think this is technically more efficient. Either way, may as well use it for good style.